src/symbols.js

```diff
@@ -19,7 +19,7 @@
 
 import type {Mode} from "./types";
 
-type Font = "main" | "ams";
+type Font = string; // can be "main" | "ams" | any other injected font by plugins
```

Comment:
It will not work until user defines a `font-face` and adds appropriate font metrics to use a new font.   
I will revert this as we have to deal with this issue in #1269. These types are not much useful until we add katex libdef to [flow-typed](https://github.com/flowtype/flow-typed) project or use something like [.js.flow](https://github.com/flowtype/flow-typed/wiki/FAQs#what-is-the-preferred-way-for-a-module-to-provide-its-own-libdef) files when they got stable.
After deleting this I don't think we need any new tests in this PR.